packages/block-library/src/page-list/edit.js

```diff
@@ -173,9 +176,50 @@ export default function PageListEdit( {
 		}
 	}, [ clientId, blockList ] );
 
+	const { replaceBlock, selectBlock } = useDispatch( blockEditorStore );
+
+	const { parentNavBlockClientId } = useSelect( ( select ) => {
+		const { getSelectedBlockClientId, getBlockParentsByBlockName } =
+			select( blockEditorStore );
+
+		const _selectedBlockClientId = getSelectedBlockClientId();
+
+		return {
+			parentNavBlockClientId: getBlockParentsByBlockName(
+				_selectedBlockClientId,
+				'core/navigation',
+				true
+			)[ 0 ],
+		};
+	}, [] );
+
 	return (
 		<>
 			<InspectorControls>
+				{ isNavigationChild && (
+					<PanelBody title={ __( 'Customize this menu' ) }>
+						<p id={ 'wp-block-page-list-modal__description' }>
+							{ __(
+								'This menu is automatically kept in sync with pages on your site. You can manage the menu yourself by clicking customize below.'
```

Comment:
Is it worth extracting this paragraph to a shared component, so that we only have one copy?